src/js/ripple/amount.js

```diff
@@ -1,12 +1,16 @@
+ 'use strict';
+
+ /*eslint-disable new-cap*/
```

Comment:
It's unlikely that it's being used externally but let's not forget to add it to the release notes. I'll change Amount.NaN() to new Amount(NaN) in this file